### PR TITLE
Reuse velocity solution

### PR DESCRIPTION
**Pull request by oguevremont**
# Description of the problem

- It was difficult to couple problems where fluid dynamics reach pseudo-steady state much faster than auxiliary physics.

# Description of the solution

- Using average velocity field for heat transport advection was already possible, but it required keeping fluid dynamics active and it was only possible for heat transport.
- A modification was made that allows using average velocity field even with fluid dynamics disabled.
- It was made possible to do this with Tracer and VOF auxiliary physics.

# How Has This Been Tested?

- A short script running two simulations, one with and one without fluid dynamics, was used.
- This simple case will be made into an example.

# Documentation

- Documentation was adjusted in Multiphysics parameters.

# Future changes

- The simple case will be made into an example.

### CODE REVIEW

File: doc/source/parameters/cfd/multiphysics.rst [hunk 1]
```diff
@@ -5,19 +5,20 @@ This subsection defines the multiphysics interface of Lethe and enables the solu
...
 
     # Tracer
-    set tracer 			= false
+    set tracer = false
 
     # Multiphase flow
-    set VOF 			= false
+    set VOF = false
+
+    set use time average velocity field = false
```
Comment (blaisb) [inline doc/source/parameters/cfd/multiphysics.rst:21]: why change the alignment here?

File: source/solvers/navier_stokes_base.cc [hunk 2]
```diff
@@ -565,23 +565,34 @@ NavierStokesBase<dim, VectorType, DofsType>::iterate()
...
+          present_solution = this->local_evaluation_point;
+        }
+      else
+        {
+          // We get the solution via an initial condition
+          this->simulation_parameters.initial_condition->uvwp.set_time(
+            this->simulation_control->get_current_time());
+          set_initial_condition_fd(
+            this->simulation_parameters.initial_condition->type);
+        }
```
Comment (blaisb) [inline source/solvers/navier_stokes_base.cc:592]: Nice

**Review by blaisb — CHANGES_REQUESTED** [review]
Cool stuff. it would be great to have a quick application_tests to ensure the feature remains stable.

**Comment by blaisb** [comment]
@oguevremont Seems like this breaks a regular restart feature?

File: doc/source/parameters/cfd/multiphysics.rst (hunk 1 above)
Comment (oguevremont) [reply doc/source/parameters/cfd/multiphysics.rst:21]: I ran this subsection in prmindent and copied the output. I can put it back the way it was though, the alignment was fine.

**Comment by oguevremont** [comment]
> @oguevremont Seems like this breaks a regular restart feature?

I fixed it

**Pull request merged**